go/obscuronode/enclave/rawdb/accessors_receipts.go

```diff
@@ -0,0 +1,202 @@
+package rawdb
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethdb"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/obscuronet/obscuro-playground/go/log"
+)
+
+// HasReceipts verifies the existence of all the transaction receipts belonging
```

Comment:
@tudor-malene As above, I think we should make it clear using a comment that this is code copied from Geth.